bloom/ros_gz/README.md

```diff
@@ -55,6 +55,19 @@ package name.
     - Usage: `$ rename-ros_gz-pkgs.bash <desired_gz_version> <space separted list of rosdistros to release>`
     - Example: `$ rename-ros_gz-pkgs.bash garden humble`
 
+The script supports to inject a custom RELEASE_REPO_URL that points to a bloom gbp
+repository different than https://github.com/gazebo-release/ros_ign-release.
```

Comment:
This is just an example right? The repo is not supposed to exist already

Reply:
The repo exists and host humble metadata for Garden. See the tabel in line 30 in this same document.